ci/gpu/build.sh

```diff
@@ -282,6 +282,10 @@ conda list
 gpuci_logger "Python py.test for cuDF"
 py.test -n 8 --cache-clear --basetemp="$WORKSPACE/cudf-cuda-tmp" --ignore="$WORKSPACE/python/cudf/cudf/benchmarks" --junitxml="$WORKSPACE/junit-cudf.xml" -v --cov-config="$WORKSPACE/python/cudf/.coveragerc" --cov=cudf --cov-report=xml:"$WORKSPACE/python/cudf/cudf-coverage.xml" --cov-report term --dist=loadscope tests
 
+gpuci_logger "Python py.tests for cuDF with spilling (CUDF_SPILL_DEVICE_LIMIT=1)"
+# Due to time concerns, we only run a limited set of tests
+CUDF_SPILL=on CUDF_SPILL_DEVICE_LIMIT=1 py.test -n 8 --cache-clear --basetemp="$WORKSPACE/cudf-cuda-tmp" --ignore="$WORKSPACE/python/cudf/cudf/benchmarks" -v --cov-config="$WORKSPACE/python/cudf/.coveragerc" --cov-append --cov=cudf --cov-report=xml:"$WORKSPACE/python/cudf/cudf-coverage.xml" --cov-report term --dist=loadscope tests/test_binops.py tests/test_dataframe.py tests/test_buffer.py tests/test_onehot.py tests/test_reshape.py
```

Comment:
We need to run the tests twice - one with `CUDF_SPILL=off` and one `CUDF_SPILL=on`. Do you know if it is possible to "parametrize" a whole module?

Reply:
Yes, that is the second link in my message above. You do something like this:
```
import pytest
pytestmark = pytest.mark.spilling
```
in each module that you want to include. Then, to the config file add a custom marker like this:
```
[pytest]
markers =
    spilling: mark a benchmark that cannot be run with pandas
```

Reply:
Yes, but we need to run the tests **twice** - one with `CUDF_SPILL=off` and one `CUDF_SPILL=on`. Or do you think we should only run using `CUDF_SPILL=on`?

Reply:
Oh I see where the confusion lies. I'm not saying to get rid of the second invocation of pytest, I think that is fine. I just don't want the sole source of truth on which tests must be run with spilling to be in the CI scripts. I want developers in the future to be able to select that subset if they need to run the tests locally. pytest lets you select tests by mark: `pytest -m $MARK`. All I'm suggesting to do is to mark the relevant modules, then change this second invocation to use that syntax instead of manually listing all the module names `pytest test_a.py test_b.py...`. That is much easier for devs to use locally.

Reply:
Aha, now I get it :)
Done